deluge/ui/web/server.py

```diff
@@ -69,6 +69,17 @@
 )
 
 
+def remove_version_header(headers):
```

Comment:
Instead of a major addition we could override the server version manually before creating the site:
```diff
+ server.version = 'TwistedWeb'
self.site = server.Site(self.top_level)
```
Since this is only set once when the server module is imported it should not be altered.

Reply:
hmm, good idea.. Done.